app/controllers/concerns/not_found.rb

```diff
@@ -0,0 +1,9 @@
+module NotFound
+  extend ActiveSupport::Concern
+
+  included do
+    def not_found
```

Comment:
The `included` block of a concern is to specify method calls to be executed when the concern gets included.

(Instance) method definitions just go in the body of the module.